src/main/daml/Daml/Finance/Instrument/Generic/Election.daml

```diff
@@ -75,13 +75,13 @@ template ElectionFactory
       -- ^ A human readable description of the election claim.
     claim : C
       -- ^ the elected sub-tree
-    observers : Observers
+    observers : Parties
```

Comment:
I wonder if we should distinguish two cases better by renaming `Observers` and `Parties`? Maybe rename `Observers` to `ObserversWithContext` (or `ParllelObservers`), and use a type synonym called `Observers` instead of `Parties`? But I'm fine to leave it as is too.

Reply:
It's a fair point but `Parties` can be used for other contexts outside of observers. Ie., `locker : Parties` on the `Lockable` interface. 

An alternative idea would be to remove the `Observer` work and replace it with something like `MappedParties` or something like that. We can address this point in the issue I raised to review the `Observers` type implementation.